dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/VersionRestRepository.java

```diff
@@ -52,6 +98,90 @@ public VersionRest findOne(Context context, Integer id) {
+        boolean isAdmin = authorizeService.isAdmin(context);
+        boolean canCreateVersion = configurationService.getBooleanProperty("versioning.submitterCanCreateNewVersion");
+        boolean isBlockEntity = StringUtils.isNotBlank(stringBlockEntity) ? Boolean.valueOf(stringBlockEntity) : true;
+
+        if (!isAdmin && !(canCreateVersion && Objects.equals(submitter, context.getCurrentUser())) ||
+            (hasEntityType  && isBlockEntity)) {
+            throw new DSpaceForbiddenException("");
+        }
+
+        WorkflowItem workflowItem = null;
+        WorkspaceItem workspaceItem = null;
+        VersionHistory versionHistory = versionHistoryService.findByItem(context, item);
+        if (Objects.nonNull(versionHistory)) {
+            Version lastVersion = versionHistoryService.getLatestVersion(context, versionHistory);
+            if (Objects.nonNull(lastVersion)) {
+                workflowItem = workflowItemService.findByItem(context, lastVersion.getItem());
+                workspaceItem = workspaceItemService.findByItem(context, lastVersion.getItem());
+            }
+        } else {
+            workflowItem = workflowItemService.findByItem(context, item);
+            workspaceItem = workspaceItemService.findByItem(context, item);
+        }
+
+        if (Objects.nonNull(workflowItem) || Objects.nonNull(workspaceItem)) {
+            throw new UnprocessableEntityException("");
+        }
+
+        Version version = StringUtils.isNotBlank(summary) ?
+                          versioningService.createNewVersion(context, item, summary) :
+                          versioningService.createNewVersion(context, item);
+        return converter.toRest(version, utils.obtainProjection());
+    }
+
+    @Override
+    @PreAuthorize("hasPermission(#versionId, 'version', 'ADMIN')")
+    protected void patch(Context context, HttpServletRequest request, String apiCategory, String model,
+                         Integer versionId, Patch patch) throws AuthorizeException, SQLException {
+        for (Operation operation : patch.getOperations()) {
+            if (!operation.getPath().equals("/summary")) {
+                throw new UnprocessableEntityException("The provided property:"
+                          + operation.getPath() + " is not supported!");
+            }
+            Version version = versioningService.getVersion(context, versionId);
+            if (Objects.isNull(version)) {
+                throw new ResourceNotFoundException("This given id:" + versionId + " does not resolve to a Version");
+            }
+            switch (operation.getOp()) {
+                case "remove":
+                    version.setSummary(StringUtils.EMPTY);
+                    break;
+                case "add":
+                    if (StringUtils.isNotBlank(version.getSummary())) {
+                        throw new DSpaceBadRequestException("The 'summary' property is setted with value:"
```

Comment:
Perhaps rephrase into something like:
The 'summary' already contains the value {value}, it is not possible to add a new value.